util.go

```diff
@@ -359,3 +359,37 @@ func addwarningtext(md *MntDir, r []rune) {
 	default:
 	}
 }
+
+const quoteChar = '\''
+
+func needsQuote(s string) bool {
+	for i := 0; i < len(s); i++ {
+		c := s[i]
+		if c == quoteChar || c <= ' ' { // quote, blanks, or control characters
+			return true
+		}
+	}
+	return false
+}
+
+// Quote adds single quotes to s in the style of rc(1) if they are needed.
+// The behaviour should be identical to Plan 9's quote(3).
+func quote(s string) string {
+	if s == "" {
+		return "''"
+	}
+	if !needsQuote(s) {
+		return s
+	}
+	b := make([]byte, 0, 10+len(s)) // Enough room for few quotes
```

Comment:
Isn't this an ideal application for a `strings.Builder` as it would save the copy in line 394? Maybe I'm missing something.